### PR TITLE
Fix Todo example Deps

### PR DESCRIPTION
**Pull request by igordmn**
task `gradlew jsBrowserDevelopmentRun` didn't work

### CODE REVIEW

**Comment by igordmn** [comment]
сс @arkivanov

**Comment by igordmn** [comment]
Fixes https://github.com/JetBrains/compose-jb/issues/1606

File: examples/todoapp/buildSrc/gradle.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
+# TODO can we get rid of duplication with root gradle.properties?
```
Comment (arkivanov) [inline examples/todoapp/buildSrc/gradle.properties:1]: Perhaps migrating to "Version catalogs" may help: https://docs.gradle.org/current/userguide/platforms.html

File: examples/todoapp/gradle.properties [hunk 2]
```diff
@@ -22,3 +22,6 @@ kotlin.code.style=official
 org.gradle.parallel=true
 org.gradle.caching=true
 kotlin.native.disableCompilerDaemon=true
+
+kotlin.version=1.6.10
```
[the remark this answers is not in the document]
Comment (igordmn) [reply examples/todoapp/gradle.properties:26]: Yes, but this change also simplifies accessing Deps.

File: examples/todoapp/buildSrc/gradle.properties (hunk 1 above)
Comment (igordmn) [reply examples/todoapp/buildSrc/gradle.properties:1]: But it is not easy to override the version in version catalog via an argument in command line.

**Comment by arkivanov** [comment]
In this case LGTM

**Pull request merged**